templates/default/config/siteConfig.js

```diff
@@ -16,6 +16,7 @@ const defaultConfig = {
   // if you have your notes in another (external) directory,
   // /content dir should be a symlink to that directory
   content: "content",
+  contentExclude: ["fake"],
```

Comment:
I guess this is some test value we don't need?

Reply:
Indeed :)